README.md

```diff
@@ -322,6 +322,17 @@ preventCollision: ?boolean = false;
 // onDragStart attribute is required for Firefox for a dragging initialization
 // @see https://bugzilla.mozilla.org/show_bug.cgi?id=568313
 isDroppable: ?boolean = false
+// Defines which resize handles should be rendered
+// Allows for any combination of:
+// 's' - South handle (bottom-center)
```

Comment:
maybe it would be better to use full forms: `bottom-center` instead of `s` and so on. We should set these settings only once, so it's not so hard to use full forms instead of shortcuts but it's a lot more intuitive (in my opinion)